docs/modules/ROOT/pages/how-tos/installation.adoc

```diff
@@ -4,6 +4,13 @@
 
 This component requires https://github.com/appuio/component-appuio-cloud-reporting[component-appuio-cloud-reporting] and is installed into the same namespace.
 This is required for this component to be able to access the billing database and its connection secrets.
+It also requires an Exoscale IAMKey and a Kubernetes/OpenShift Service Account token in the target cluster to get `buckets.exoscale.crossplane.io` resources.
+
+== Sources
+
+The data is matched from k8s cluster and Exoscale organization.
+The Kubernetes Service Account token is required to have `get` permissions on `Namespaces` and `Buckets`(either exoscale or cloudscale) resources.
```

Comment:
Not sure what you meant. On both cloud providers, the bucket resource is of kind `buckets.exoscale.crossplane.io`
```suggestion
The Kubernetes Service Account token is required to have `get` permissions on `Namespaces` and `buckets.exoscale.crossplane.io` resources.
```

Reply:
Yes it will work only with exoscale buckets, updating the code...